### DISCUSSION TITLE
Best way to start a workflow without using PromptTask?

### DISCUSSION THREAD

**Discussion by benji-ai** [griptape-ai/griptape#513] [Q&A] [answered]
Apologies if this is the wrong place to ask a question - I'm newish to programming and Github. Thanks in advance for help!

Question: what's the optimal way to start a workflow without using PromptTask? I'm attempting to use BaseTask to start a workflow by printing a comment of "starting workflow," but can't get it to work.

For context, I'm following along with this tutorial: https://learn.griptape.ai/latest/courses/compare-movies-workflow/04_adding_flexibility/ and I can't get it to work (the example code doesn't work, either). I believe I need to create a start_task, hence why I'm reaching out. 

Note that in the example code provided from the tutorial, start_task doesn't appear to be defined. 

```
from dotenv import load_dotenv

from griptape.structures import Workflow
from griptape.tasks import PromptTask

load_dotenv()

workflow = Workflow()

#<---- this is where I'd like to create a "dummy" start_task without invoking the prompt

movie_descriptions = [
    "A boy discovers an alien in his back yard",
    "A shark attacks a beach",
    "A princess and a man named Wesley"
]

end_task = PromptTask("""
    How are these movies the same:
    {% for value in parent_outputs.values()%}
    {{ value }}
    {% endfor %}
    """,
    id="END")

for description in movie_descriptions:
    movie_task = PromptTask(
        "What movie title is this? Return only the movie name: {{ description }} ",
        context={"description": description})
    workflow.insert_tasks(start_task, [movie_task], end_task) #<-- this is where start_task is referenced

workflow.run()

```

**Comment by shhlife** [comment]
Hi @benji-ai - so sorry I didn't see this earlier! Feel free to chat with us in our Discord channel - that's a great place to ask questions to the team: https://discord.gg/9hWfamRPS6


Thank you for pointing out the start_task is missing! It used to be there, so I'll have to figure out how ti disappeared - I'll fix it right now!

**Comment by shhlife** [comment, marked as answer]
I've added back the code, and you were absolutely correct with the location you wanted to put the start task.

It would look something like:
```
# Create tasks
start_task = PromptTask("I will provide you a list of movies to compare.", id="START")
```

I've updated the site! Thanks again :)
-Jason
